lib/ddtrace/contrib/httparty/patcher.rb

```diff
@@ -0,0 +1,29 @@
+module Datadog
+  module Contrib
+    module HTTParty
+      # Patcher enables patching of 'httparty' module.
+      module Patcher
+        include Contrib::Patcher
+
+        module_function
+
+        def patched?
+          done?(:httparty)
+        end
+
+        def patch
+          do_once(:httparty) do
+            require 'ddtrace/contrib/httparty/module_patch'
+            require 'ddtrace/contrib/httparty/request_patch'
```

Comment:
I quite like these `require` statements being delayed up until this point! 💯 

It's definitely something we could do in other parts of the code.

Reply:
I think I copied this from another existing integration :)